src/components/Auth/SignIn.tsx

```diff
@@ -33,19 +55,44 @@ const SignIn = () => {
   }, [])
 
   const {
-    login: { mutateAsync: login, isError, error, isPending },
+    login: { mutateAsync: login, isError, error },
   } = useAuth()
   const [verifyNeeded, setVerifyNeeded] = useState(false)
+  const { mutateAsync: checkVerificationCodeStatus } = useVerificationCodeStatus()
+  const resendVerificationCode = useResendVerificationCode()
```

Comment:
Why are the two above declarations different since they are being used in a similar manner later on?